bin/start_fe.sh

```diff
@@ -143,10 +141,28 @@ if [ x"$HOST_TYPE" != x"" ]; then
     HOST_TYPE="-host_type $HOST_TYPE"
 fi
 
-if [ ${RUN_DAEMON} -eq 1 ]; then
-    nohup $LIMIT $JAVA $final_java_opt com.starrocks.StarRocksFE ${HELPER} ${HOST_TYPE} "$@" >> $LOG_DIR/fe.out 2>&1 </dev/null &
+LOG_FILE=$LOG_DIR/fe.out
+
+if [ ${RUN_LOG_CONSOLE} -eq 1 ] ; then
+    if [ ! -w $STARROCKS_HOME/conf/fe.conf ] ; then
+        # workaround configmap readonly, can't change its content
```

Comment:
Why not `cp $STARROCKS_HOME/conf/fe.conf $STARROCKS_HOME/conf/fe.conf.readonly`?

Reply:
fe.conf is a symbol link, cp fe.conf to fe.conf.readonly, doesn't make fe.conf writable.